### PR TITLE
Testing CircleCI with dummy route tests

### PR DESCRIPTION
**Pull request by farhanali3193**
@tchitrakorn @jabrake 
1. Found a way for circleCI to work with route tests and not connect to the local db. However, will need to refactor since I do plan to use a test db.
2. DO NOT MERGE YET. Will add a test db to fix this error. Thanks

### CODE REVIEW

File: pg/index.js [hunk 1]
```diff
@@ -1,11 +1,15 @@
 const { Client } = require('pg');
+const database = process.env.NODE_ENV === 'development' ? 'qna' : 'qna-test'
```
Comment (tchitrakorn) [inline pg/index.js:2]: Would love to chat more about NODE_ENV at some point if you don't mind!

File: test/routes/routes.test.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+beforeAll(async () => {
+  db.client.connect()
+    .then(() => console.log('TEST PG CONNECTED'))
+    .catch((err) => console.log('TEST PG NOT CONNECTED', err))
+})
+
+afterAll(async () => {
+  await db.client.end();
+});
```
Comment (tchitrakorn) [inline test/routes/routes.test.js:16]: If you have some time, could you please clarify why you need to do connect to the database again here? I think someone had tried to explained this earlier but I didn't quite catch it. Really curious if this has to do with one of the many problems that I have with CircleCI.

File: test/routes/routes.test.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+describe('GET/qa/questions', () => {
+  it('responds with 200 status code', async () => {
+    let response = await request.get('/qa/questions?product_id=1&count=1')
+    // console.log('RESPONSE', response)
+    expect(response.statusCode).toBe(200)
+  }, 1000 * 30);
```
Comment (tchitrakorn) [inline test/routes/routes.test.js:23]: I'm assuming 1000 * 30 is the modified timeout for jest (or supertest)? If that's the case, I also had to do something similar for mine. Really curious how we could improve this number!

**Review by tchitrakorn — APPROVED** [review]
Hi @farhanali3193, I think it looks great! I've left a couple of comments (again, for my own understanding of things) so I will just go ahead and merge this one for you :)
Edit: I noticed that your CircleCI failed but if I recalled correctly, you mentioned currently working off this version to get CircleCI to work so again I will just go ahead and merge!

File: pg/index.js (hunk 1 above)
Comment (farhanali3193) [reply pg/index.js:2]: Sure.

File: test/routes/routes.test.js (hunk 2 above)
Comment (farhanali3193) [reply test/routes/routes.test.js:16]: @tchitrakorn I should have removed this piece of code. Thanks for pointing it out. I had to do this to make CircleCI work before I added the postgres image in the config.yml file. I will remove it in my next PR.

File: test/routes/routes.test.js (hunk 3 above)
Comment (farhanali3193) [reply test/routes/routes.test.js:23]: @tchitrakorn 5 seconds is the default wait time for jest. By improving this number, do you mean how to reduce the time it takes to get a response from the server?

**Pull request merged**